storm-core/src/jvm/org/apache/storm/trident/windowing/WindowKryoSerializer.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.storm.trident.windowing;
+
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import org.apache.storm.serialization.SerializationFactory;
+
+import java.nio.ByteBuffer;
+import java.util.Map;
+
+/**
+ * Kryo serializer/deserializer for values that are stored as part of windowing. This can be used in {@link WindowsStore}.
+ * This class is not thread safe.
+ *
+ */
+public class WindowKryoSerializer {
```

Comment:
It would be nice to change the name of this class to something more descriptive of its function, since the class itself has nothing to do with windowing.